app/uk/gov/hmrc/plasticpackagingtax/registration/views/group/member_name_page.scala.html

```diff
@@ -62,15 +62,16 @@
 }
 
 @govukLayout(
-    title = Title("contactDetails.member.name.title", organisationName),
+    title = Title("contactDetails.member.name.heading", organisationName),
```

Comment:
This looks to expect a parameterised message but `contactDetails.member.name.heading` does not contain a placeholder.

Reply:
In fact (see comment below), why has this heading been changed? The ticket nor the prototype use this heading.

Reply:
Heading added omit parameterised details from the tab title. I'll need to remove [**, organisationName),**]